internal/hardwaremanager/controller/hostfirmwarecomponents_manager_test.go

```
@@ -939,192 +939,150 @@ var _ = Describe("HostFirmwareComponents Manager", func() {
-
-			// validateOnly=false: returns error for invalid firmware spec
-			validateOnly = false
-			required, err = IsFirmwareUpdateRequired(ctx, fakeClient, logger, bmh, spec, validateOnly)
-			Expect(err).To(HaveOccurred())
-			Expect(err.Error()).To(ContainSubstring("missing BIOS firmware URL"))
-			Expect(required).To(BeFalse())
-		})
+		const testNS = "test-namespace"
+
+		createCatalogAndProfile := func(
+			biosFw, bmcFw string,
```

golangci-lint (2.12.2)

[error] init$6$12$1 - biosFw is unused

(unparam)